src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/MessageboxInstancesController.cs

```diff
@@ -174,7 +183,7 @@ public async Task<ActionResult> GetMessageBoxInstanceList(int instanceOwnerParty
 
             List<Instance> allInstances = queryResponse.Instances;
 
-            allInstances.RemoveAll(i => i.VisibleAfter > DateTime.UtcNow || i.Status.IsHardDeleted);
+            allInstances.RemoveAll(i => i.VisibleAfter > DateTime.UtcNow);
```

Comment:
Is the removel of permanently deleted instances now implicit some where else?

Reply:
It is placed in instanceRepository as it is to cover both this controller and the messageboxController. 
https://github.com/Altinn/altinn-studio/blob/031aefc0044e81d914bce7fd63953e98bf6549ad/src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/InstanceRepository.cs#L125

Reply:
oh, a previous change. I should have looked.